routes/author/index.js

```diff
@@ -99,10 +108,10 @@ module.exports = function(req, res){
         console.log(err);
       }
       books = encodeImgSrc(books);
-      var title = author.name + "先生のKindle化された著書";
-      res.render( 'author', {
-        title : title,
-        description: title + "の一覧ページです",
+      const title = `${author.name}先生のKindle化された著書`;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

*  Unexpected '`'.